jdatetime/__init__.py

```diff
@@ -531,131 +562,67 @@ def __format__(self, format):
         """
         return self.strftime(format)
 
-    def strftime(self, format):
-        """format -> strftime() style string."""
-        # TODO: change stupid str.replace
-        # formats = {
-        #           '%a': lambda: self.j_weekdays_short[self.weekday()]
-        # }
-        # find all %[a-zA-Z] and call method if it in formats
-
-        # convert to unicode
-        try:
```

Comment:
this conversion is removed from the code, is it intentional? maybe is safer to keep it for backward compatibility and remove it in a later PR if needed?

Reply:
It was by mistake. and I didn't realize it because we don't have a test to cover this case.
I've reverted back it part and also added a test for it.